src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/ProvideSemanticTokensDeltaParams.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using OmniSharp.Extensions.LanguageServer.Protocol.Models.Proposals;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+{
+#pragma warning disable CS0618 // Type or member is obsolete
+    internal class ProvideSemanticTokensDeltaParams : SemanticTokensDeltaParams
```

Comment:
Maybe leaving a note that the version has moved to a different namespace would be helpful for clarity

Reply:
Not sure I really understood this comment. If the change I made to remove the duplicated class' didn't resolve it could you go into more detail?

Reply:
Ah the duplicated class was my confusion, all good now, thanks!